### DISCUSSION TITLE
How to update  items values for suggestion in Mention while Editing?

### DISCUSSION THREAD

**Discussion by bannarisoftwares** [ueberdosis/tiptap#6135] [General Questions]
### What problem are you facing?

How to update items values for suggestion in Mention extension?

### What’s the solution you would like to see?

I am not able to update the items values for suggestion in Mention extension while editing in tiptap editor.

### What alternatives did you consider?

I tried to add callback function and passed into suggestion, but the values are still points to old item values.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by Nem-Tudo** [comment]
Same problem

**Comment by andrictham** [comment]
@Nem-Tudo @bannarisoftwares Any of you figure this out?

**Comment by bannarisoftwares** [comment]
No @andrictham

**Comment by geekyayush** [comment]
Has anyone figured it out? I am facing the same issue.

How is this use case not covered?

**Comment by geekyayush** [comment]
Here, found the solution: https://codesandbox.io/s/suggestion-mention-update-list-xypnd3

By this awesome guy: https://discord.com/channels/818568566479257641/1050070455983296512/1055948369983443014

**Comment by ideosyncretic** [comment]
@geekyayush Adding an arbitrary property to `editorProps` seems a bit iffy, especially when using Typescript.

@andrictham and I got it working with Storage (and Typescript in our project, not included in this example): https://codesandbox.io/p/sandbox/tiptap-mentions-with-storage-6cxxyz

https://tiptap.dev/docs/editor/guide/custom-extensions#storage

The downside is having to coerce/cast the custom extension storage type:
https://tiptap.dev/docs/editor/guide/typescript#storage-types

**Reply by kresimir-coko** [reply]
Very clean solution 👍
